extensions/amp-ad-network-dianomi-impl/0.1/amp-ad-network-dianomi-impl.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpA4A} from '../../amp-a4a/0.1/amp-a4a';
+import {userAssert} from '../../../src/log';
+
+export class AmpAdNetworkDianomiImpl extends AmpA4A {
+  /** @override */
+  getAdUrl() {
+    const paramId = this.element.getAttribute('data-request-param-id');
+    const isValid = new RegExp(/^.[0-9]*$/);
```

Comment:
Extract this regex as a constant, which will also prevent from creating a new regex object every time this is called. 

```
const REQUEST_PARAM_ID = /^.[0-9]*$/;
```
On lin 26:
```
REQUEST_PARAM_ID.test(paramId)
```

Reply:
I have moved the regex to be a contant.